src/vulkan/device.cc

```diff
@@ -24,6 +25,23 @@ namespace amber {
 namespace vulkan {
 namespace {
 
+const std::vector<const char*> kRequiredValidationLayers = {
```

Comment:
This will case issues with exit destructors. Change it to a

const char* kRequiredValidationLayers[] = { ... }

Reply:
done